llama-index-core/llama_index/core/indices/property_graph/transformations/schema_llm.py

```diff
@@ -47,62 +47,37 @@
     "HAS_ALIAS",
 ]
 
-# Which entities can be connected to which relations
-DEFAULT_VALIDATION_SCHEMA: Dict[str, Any] = {
-    "PRODUCT": (
-        "USED_BY",
-        "USED_FOR",
-        "LOCATED_IN",
-        "PART_OF",
-        "WORKED_ON",
-        "HAS",
-        "IS_A",
-    ),
-    "MARKET": ("LOCATED_IN", "PART_OF", "WORKED_ON", "HAS", "IS_A"),
-    "TECHNOLOGY": (
-        "USED_BY",
-        "USED_FOR",
-        "LOCATED_IN",
-        "PART_OF",
-        "WORKED_ON",
-        "HAS",
-        "IS_A",
-    ),
-    "EVENT": ("LOCATED_IN", "PART_OF", "WORKED_ON", "HAS", "IS_A"),
-    "CONCEPT": ("USE_BY", "USED_FOR", "PART_OF", "WORKED_ON", "HAS", "IS_A"),
-    "ORGANIZATION": ("LOCATED_IN", "PART_OF", "HAS", "IS_A"),
-    "PERSON": (
-        "BORN_IN",
-        "DIED_IN",
-        "LOCATED_IN",
-        "PART_OF",
-        "WORKED_ON",
-        "HAS",
-        "IS_A",
-    ),
-    "LOCATION": (
-        "LOCATED_IN",
-        "PART_OF",
-        "HAS",
-        "IS_A",
-        "DIED_IN",
-        "BORN_IN",
-        "USED_BY",
-        "USED_FOR",
-    ),
-    "TIME": ("BORN_IN", "DIED_IN", "LOCATED_IN", "PART_OF", "HAS", "IS_A"),
-    "MISCELLANEOUS": (
-        "USED_BY",
-        "USED_FOR",
-        "LOCATED_IN",
-        "PART_OF",
-        "WORKED_ON",
-        "HAS",
-        "IS_A",
-        "BORN_IN",
-        "DIED_IN",
-    ),
-}
+# Convert the above dict schema into a list of triples
+Triple = Tuple[str, str, str]
+DEFAULT_VALIDATION_SCHEMA: List[Triple] = [
```

Comment:
Just wanted to clarify that this list is based on my judgment only - it's only when you begin trying to identify valid triples that you realize how nonsensical some of the node/node pairs are in the prior JSON structure. If there's a guideline on what a good default validation schema would be, happy to make the changes - however, I think that no realistic case would rely on this default schema, so it doesn't need to be exhaustive.

Reply:
Yea the default isn't really meant to be universally useful -- more so a way to get started. I'm fine with this list